components/kyma-environment-broker/internal/process/provisioning/cm_kymaversion_configurator.go

```diff
@@ -0,0 +1,53 @@
+package provisioning
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type ConfigMapKymaVersionConfigurator struct {
```

Comment:
Maybe this could be named `KymaVersionConfigurator`? Not sure if this needs to reveal the implementation details like ConfigMap storage.

Reply:
ok, changed